script.js

```diff
@@ -2,53 +2,64 @@ const $books = document.querySelector('.books');
 const $title = document.querySelector('.title');
 const $author = document.querySelector('.author');
 const $addButton = document.querySelector('.addButton');
-const bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+const $error = document.querySelector('.error');
 
 document.addEventListener('DOMContentLoaded', () => {
-  function remove(idSelected) {
-    document.getElementById(idSelected).parentElement.remove();
-    bookList.splice(bookList.findIndex((book) => book.title + book.author === idSelected), 1);
-    localStorage.setItem('bookList', JSON.stringify(bookList));
-  }
+  class Books {
+    constructor() {
+      this.bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+      if (this.bookList.length > 0) {
+        this.bookList.forEach((book) => {
+          $books.insertAdjacentHTML(
+            'beforeend',
+            `
+            <div class="itemContainer"><p>"${book.title}" by ${book.author}</p><button type="button" class="remove" id="${book.title}${book.author}">Remove</button></div>
+          `,
+          );
+          const $removeButton = document.getElementById(`${book.title}${book.author}`);
+          $removeButton.addEventListener('click', () => {
+            this.remove(`${book.title}${book.author}`);
+          });
+        });
+      }
+    }
 
-  function add(title, author) {
-    const item = { title, author };
-    item.title = title;
-    item.author = author;
-    bookList.push(item);
-    $books.insertAdjacentHTML(
-      'beforeend',
-      `
-        <span><p>${item.title}<br>${item.author}</p><button type="button" id="${item.title}${item.author}">Remove</button><hr></span>
-      `,
-    );
-    localStorage.setItem('bookList', JSON.stringify(bookList));
-    const $removeButton = document.getElementById(`${item.title}${item.author}`);
-    $removeButton.addEventListener('click', () => {
-      remove(`${item.title}${item.author}`);
-    });
-  }
+    remove(idSelected) {
+      document.getElementById(idSelected).parentElement.remove();
+      this.bookList.splice(this.bookList.findIndex((e) => e.title + e.author === idSelected), 1);
```

Comment:
## [Optional]

- [ ] Please use the Slice method instead of Splice because it doesn't mutate the array and it is a good practice to use methods that don't mutate the array. 👍